sdk/src/lib.rs

```diff
@@ -774,20 +774,6 @@ impl<T> WfExitValue<T> {
     }
 }
 
-/// Activity functions may return these values when exiting
-pub enum ActExitValue<T> {
```

Comment:
This needed to move to prevent orphan issues with the `ActivityOutput` trait impls. Seemed like an okay price to pay if it makes the rest of this work out...